### DISCUSSION TITLE
Dock used all my mechine's disk

### DISCUSSION THREAD

**Discussion by pggdt** [plausible/analytics#1220] [Self-Hosted Support] [answered]
Hi, I use Plausible for several months and it worked great. But I found my disk is full and I couldn't figure out how to delete some old data to release some space. I have only one docker running on my machine(plausible),it used 33G disk space. Thank you for help.

**Comment by pggdt** [comment, marked as answer]
[How to access the postgres and clickhouse database in self hosted version](https://github.com/plausible/analytics/discussions/621) This might help

**Comment by pggdt** [comment]
Here comes the quick and silly way[caution: may cause data lose, do it with your own risk]:
As I'm not a postgres nor clickhouse nor docker professor, I know nothing about them. So I use 
`du -hsx * | sort -rh`
 and found that the most biggest dir is in 
`/var/lib/docker/volumes/plausible_event-data/_data/store`
,then I delete the biggest dir in it(16GB), after restart the docker 
`sudo docker-compose -f docker-compose.yml -f geoip/docker-compose.geoip.yml up`,
everything goes well again.

**Comment by kleuter** [comment]
Just running plausible for 3 days and **/var/lib/docker/volumes/plausible_event-data/_data/store** is already **over 800M**. Why is that? Any official solution maybe? I'd rather avoid manual deleting of files

**Comment by kleuter** [comment]
@metmarkosaric marked the question answered, but connecting to the Clickhouse database shows very small tables size actually:
![image](https://user-images.githubusercontent.com/2600624/130615445-b4b9faff-b1dd-4797-a571-d9343bdaeccf.png)
